### ISSUE TITLE
[A11yMAS][Accessibility][Keyboard][MAS2.4.3][Eclipse][Wipro]:  Keyboard focus order is illogical in Available software Window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1349]
**Repro Steps:**
**Scenario 1:**

1. Open and Start Eclipse.
2. Press "Alt+H" to open Help menu.
3. Navigate and select Install new software Menu item under Help menu.
4. In Available Software dialog , navigate to Work with text edit box and provide http://dl.microsoft.com/eclipse/ link and hit enter
5. Tab  up to "Contact all update sites during install to find required software" checkbox then press tab to "Next" button and hit enter.
6. In Install details dialog, tab to "Back" button and hit enter then again navigate upto "Details" Read only edit and press tab then observe the keyboard focus order

**Scenario 2:**
1. Open and Start Eclipse.
2. Press "Alt+H" to open Help menu.
3.Navigate and select Install new software Menu item under Help menu.
4. In Available Software dialog , navigate to Work with text edit box and provide http://dl.microsoft.com/eclipse/ link and hit enter
5. Tab to up to "Contact all update sites during install to find required software" checkbox then press tab to "Next" button and hit enter.
6.In Install details dialog, tab to "Back" button and hit enter then again navigate up to  "Contact all update sites during install to find required software" checkbox then press tab and observe the keyboard focus

**Expected Result:**
**Scenario 1:**
Keyboard focus order should be "Details" Read only Edit->More->Show only the latest available software->hide Items that are already installed->Group Items by Category->Already installed->Show only software applicable to target environment->"Contact all update sites during install to find required software"->help->Next button
**Scenario 2:** 
Keyboard focus order should be "Contact all update sites during install to find required software" checkbox->Help button->Next button

**Actual Result:**
**Scenario 1:**
Keyboard focus order should be "Details" Read only Edit->Show only the latest available software->hide Items that are already installed->Group Items by Category->Show only software applicable to target environment->"Contact all update sites during install to find required software"
Keyboard focus is not going to "More" and "Already Installed" links
**Scenario 2:** 
Two tab Loss of focus is observed i.e.: "Contact all update sites during install to find required software" checkbox->Next button

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version**: Version 1709(OS Build 16299.192)

![keyboard focus is illogical from details in available software window](https://user-images.githubusercontent.com/22956130/35389259-04e8569c-01fd-11e8-876a-bc88797cc3d4.jpg)
![keyboard focus order is not visible for two tabs in available software window](https://user-images.githubusercontent.com/22956130/35389261-0518865a-01fd-11e8-97f4-0d3afdb21060.jpg)

**Comment by Eskibear** [comment]
It is determined by Eclipse IDE. We cannot fix it.

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=534670

**Comment by Eskibear** [comment]
* **Importance of experience**: somewhat important. It's about the one-time installation.
* **Quality of experience**: not impacted. A simple workaround is available, users can install the plugin from **Help->Marketplace** instead of **Help->Install New Software**.  
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749830
